tests/deployments/helm/nginx-k8s.yaml

```diff
@@ -0,0 +1,52 @@
+---
```

Comment:
Can we put this deployment spec in test-services with the other nginx stuff?  This would be useful to have just for general development to throw up on a k8s cluster manually.

Reply:
This is actually the same spec from `tests/monitors/nginx`. I can just delete this file and update the test to use the existing spec instead.